completions/vncviewer

```diff
@@ -13,11 +13,11 @@ _vncviewer_bootstrap()
     # Install real completion for subsequent completions
     complete -F $fname vncviewer
     $fname "$@" # Generate completions once for now
-    unset -f _vncviewer_bootstrap
+    unset -f "$FUNCNAME"
 } &&
-    complete -F _vncviewer_bootstrap vncviewer
+    complete -F _comp_cmd_vncviewer__bootstrap vncviewer
 
-_tightvncviewer()
+_comp_cmd_tightvncviewer()
```

Comment:
https://github.com/scop/bash-completion/blob/3ed0e1f9bed7b7ec7950a20a60ac9345aab8f7a3/completions/vncviewer#L8-L9

These also need to be updated.

Reply:
Thanks! Addressed in a fixup commit (gitlint will fail with that until #940 is merged). Ditto the two test ignores below.